src/main/java/org/dependencytrack/notification/publisher/PublishContext.java

```diff
@@ -180,4 +190,21 @@ private static Vulnerability convert(final org.dependencytrack.model.Vulnerabili
 
     }
 
+    public record VulnerabilityUpdateDiff(String id, String source, Severity oldSeverity, Severity newSeverity) {
+
+        private static VulnerabilityUpdateDiff convert(final org.dependencytrack.model.Vulnerability notificationVuln,
+                                                       final org.dependencytrack.model.VulnerabilityUpdateDiff notificationVulnUpdateDiff) {
+            if (notificationVuln == null || notificationVulnUpdateDiff == null) {
+                return null;
+            }
+            return new VulnerabilityUpdateDiff(
+                    notificationVuln.getVulnId(),
+                    notificationVuln.getSource(),
+                    notificationVulnUpdateDiff.getOldSeverity(),
+                    notificationVulnUpdateDiff.getNewSeverity()
+            );
```

Comment:
Do not include the actual diff values in here. Subjects are intended to give only surface-level information about the notification being dispatched, and they all appear in the logs. 

Consider just mapping to `SUBJECT_VULNERABILITY`, and re-using `Vulnerability#convert` instead.